.buckconfig

```diff
@@ -0,0 +1,21 @@
+[repositories]
+root = .
+prelude = prelude
+toolchains = toolchains
+none = none
+
+[repository_aliases]
+config = prelude
+fbcode = none
+fbsource = none
+buck = none
+
+[parser]
+target_platform_detector_spec = target:root//...->prelude//platforms:default
+
+[project]
+ignore = .git
+
+[python]
```

Comment:
Is this section added by you or was the entire file generated ? I would hope this is not necessary

Reply:
This file was generated, I only added the last two lines, for the interpreter and the pyre paths.

Reply:
I have deleted the additional lines.